packages/dds/matrix/src/test/memory/sharedMatrix.spec.ts

```diff
@@ -0,0 +1,205 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { type IMemoryTestObject, benchmarkMemory } from "@fluid-tools/benchmark";
+import { MockFluidDataStoreRuntime } from "@fluidframework/test-runtime-utils/internal";
+
+import { SharedMatrix } from "../../index.js";
+import { matrixFactory, createLocalMatrix } from "../utils.js";
+
+describe("SharedMatrix memory usage", () => {
+	const matrixValue = "testValue";
```

Comment:
Nit: if this is a dummy value we use as cell data, what do you think about calling it "cellValue"?